cmd/registry/cmd/upload/bulk/bulk.go

```diff
@@ -32,5 +32,7 @@ func Command(ctx context.Context) *cobra.Command {
 
 	cmd.PersistentFlags().String("project-id", "", "Project ID to use for each upload")
 	_ = cmd.MarkFlagRequired("project-id")
+
+	cmd.PersistentFlags().Int("jobs", 10, "Number of upload jobs to run simultaneously")
```

Comment:
Originally I was thinking of doing this on the root but it seemed clearer to only have it on commands that use parallel workers. I expect that we'll want to duplicate this for all such commands.